app/assets/javascripts/components/NewsletterSticky.js

```diff
@@ -0,0 +1,200 @@
+define([
+  'jquery',
+  'DoughBaseComponent',
+  'eventsWithPromises',
+  'utilities',
+  'common'
+], function($, DoughBaseComponent, eventsWithPromises, utilities, MAS) {
+  'use strict';
+
+  var NewsletterSticky,
+    defaultConfig = {
+      appearsAfterPercentage: 40,
+      visibleClassName: 'news-signup-sticky--visible',
+      closeClassSelector: '.news-signup-sticky__close',
+      closedClassName: 'news-signup-sticky__close--closed'
+    },
+    $newsletterStickForm,
+    newsletterCloseBoxCookieURL;
+
+  NewsletterSticky = function($el, config) {
+    NewsletterSticky.baseConstructor.call(this, $el, config, defaultConfig);
+  };
+
+  /**
+   * Inherit from base module, for shared methods and interface
+   */
+  DoughBaseComponent.extend(NewsletterSticky);
+
+  NewsletterSticky.componentName = 'NewsletterSticky';
+
+  /**
+   * Initialize the component
+   * @param {Promise} initialised
+   * @return {NewsletterSticky}
+   */
+  NewsletterSticky.prototype.init = function(initialised) {
+    this._initialisedSuccess(initialised);
+    this._setVars();
+
+    this._setListeners(true);
+    this._handleScroll();
+
+    return this;
+  };
+
+  /**
+   * Sets up local variables for the sticky newsletter component
+   */
+  NewsletterSticky.prototype._setVars = function() {
+    this.closeButton = this.$el.find(this.config.closeClassSelector);
+    this.window = $(window);
+
+    $newsletterStickForm = this.$el.find('form');
+    newsletterCloseBoxCookieURL = $newsletterStickForm.find('#close-box-cookie-url').val();
+  };
+
+  /**
+   * Turn on and off the events
+   * @param {Boolean} isActive true=on false=off
+   */
+  NewsletterSticky.prototype._setListeners = function(isActive) {
+    this._setScrollListener(isActive);
+
+    if (isActive) {
+      this.closeButton.on('click', $.proxy(this._handleCloseClick, this));
+      this.$el.on('click', '.button--done', $.proxy(this._handleCloseClick, this));
+      $newsletterStickForm.on('ajax:error', $.proxy(this._formAjaxErrorHandler, this));
+    }
+    else {
+      this.closeButton.off('click', $.proxy(this._handleCloseClick, this));
+      this.$el.off('click', '.button--done', $.proxy(this._handleCloseClick, this));
+      $newsletterStickForm.off('ajax:error', $.proxy(this._formAjaxErrorHandler, this));
+    }
+  };
+
+  /**
+   * Sets the scroll event on the window on or off
+   * @param {Boolean} isActive true=on false=off
+   */
+  NewsletterSticky.prototype._setScrollListener = function(isActive) {
+    $(window)[isActive ? 'on' : 'off']('scroll.newsletter', this._scrollDebounceMethod(isActive));
```

Comment:
Can we make the event name a config variable? Just in case (I know highly highly unlikely with this) we need to run multiple instances of this component, this will clash.